server/controllers/inventory/depots/index.js

```diff
@@ -88,6 +96,21 @@ function update(req, res, next) {
   // prevent updating the uuid by accident
   if (req.body.uuid) { delete req.body.uuid; }
 
+  // handle location
+  if (req.body.hasLocation === 0) {
```

Comment:
Why can't we just send back `location_uuid : null` and that be it?  Why do we need to know `hasLocation` on the server?